cms/djangoapps/contentstore/views/item.py

```diff
@@ -1231,13 +1233,27 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
                 })
             elif xblock.category == 'sequential':
                 rules_url = settings.PROCTORING_SETTINGS.get('LINK_URLS', {}).get('online_proctoring_rules', "")
+
+                proctoring_provider = None
+                proctoring_exam_configuration_link = None
+                if xblock.is_proctored_exam:
+                    exam = get_exam_by_content_id(course.id, xblock_info['id'])
```

Comment:
How about a new function in edx_proctoring.api that returns the instructor dashboard url for the course_id/content_id? That way, the logic about choosing based on the backend can be contained in edx_proctoring.